Zend/zend_attributes.c

```diff
@@ -35,7 +35,7 @@ static zend_object_handlers attributes_object_handlers_sensitive_parameter_value
 
 static HashTable internal_attributes;
 
-void validate_attribute(zend_attribute *attr, uint32_t target, zend_class_entry *scope)
+static void validate_attribute(zend_attribute *attr, uint32_t target, zend_class_entry *scope)
```

Comment:
Makes sense, but should probably be merged separately.

Reply:
Yes, sorry this commit wasn't supposed to land here,.